src/libs/actions/ReimbursementAccount/setupWithdrawalAccount.js

```diff
@@ -135,6 +137,58 @@ function mergeParamsWithLocalACHData(data) {
     return updatedACHData;
 }
 
+function getVBBADataForOnyx() {
```

Comment:
This already exists in BankAccount.js [here](https://github.com/Expensify/App/blob/main/src/libs/actions/BankAccounts.js#L52)